gradle/gradle-mvn-push.gradle

```diff
@@ -10,7 +10,7 @@ mavenPublish {
   releaseSigningEnabled = !getGpgKey().isEmpty()
 
   targets {
-    installArchives {
+    projectArchives {
```

Comment:
is it possible to call this `installLocally`? just so its consistent with some of our other projects